src/main/resources/db/migration/0250/V286__Applications.sql

```diff
@@ -0,0 +1,48 @@
+CREATE TABLE accelerator.application_statuses (
+    id INTEGER PRIMARY KEY,
+    name TEXT NOT NULL UNIQUE
+);
+
+CREATE TABLE accelerator.applications (
+    id BIGINT PRIMARY KEY GENERATED BY DEFAULT AS IDENTITY,
+    project_id BIGINT NOT NULL REFERENCES projects ON DELETE CASCADE,
```

Comment:
We can add unique of project_id to make sure 1:1 application to project ratio

Reply:
Good call, yeah.